.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Do we need to commit this file?

Reply:
> Is this just combining the two apis, or adding some new capabilities?

just combining two apis.no new capabilities.



> Do we need to commit this file?

This file is vscode + jest debug config. This adds the ability to single-step debugging. I think it's developer friendly.

Reply:
Hmm, but I can't review the config file as I don't use it...